src/read.c

```diff
@@ -1837,11 +1837,8 @@ static avifBool avifParseFileTypeBox(avifFileType * ftyp, const uint8_t * raw, s
     if ((compatibleBrandsBytes % 4) != 0) {
```

Comment:
We could also check compatibleBrandsBytes > (4 * MAX_COMPATIBLE_BRANDS) here to disallow a very-long compatible_brands list. Not sure if that is a good idea.